engine/src/main/java/org/hibernate/search/engine/environment/classpath/spi/DefaultClassAndResourceResolver.java

```diff
@@ -19,18 +19,18 @@
  *
  * @author Hardy Ferentschik
  */
-public final class DefaultClassResolver implements ClassResolver {
+public final class DefaultClassAndResourceResolver implements ClassResolver, ResourceResolver {
```

Comment:
I'm not sure I would have use the same class for the 2 interfaces. It seems a bit confusing to me.

Not a veto, just a remark.